contracts/src/factory/facets/architect/Architect.sol

```diff
@@ -43,11 +43,17 @@ contract Architect is
   //                            Space
   // =============================================================
   function createSpace(
-    SpaceInfo calldata spaceInfo
+    SpaceInfo memory spaceInfo
   ) external nonReentrant whenNotPaused returns (address) {
     return _createSpace(spaceInfo);
   }
 
+  function createSpaceWithPrepay(
```

Comment:
I ran build-contract-types and no TS interface was updated to contain createSpace.
Does this need to be added to IArchitecht.sol?

Looking at ISpaceArchitectShim, it uses IArchitect for typings, but then uses Architect.abi.json, which is a different pattern than the other shims, where it uses typings/Xxx and Xxx.abi.json

I don't know the history of the shim - should it be converted to not use IArchitect but instead Architect?

Reply:
yep, my bad. I'll add it in